docs/Advanced-Usage/Debugging-and-Profiling-on-FPGA/Dromajo.rst

```diff
@@ -0,0 +1,112 @@
+.. _dromajo:
+
+Dromajo Co-simulation with BOOM designs
+==================================================
+
+Instead of using TracerV to provide a cycle-by-cycle trace of a target
+CPU's architectural state, you can use the `Dromajo co-simulator <https://github.com/chipsalliance/dromajo>`_ to verify
+the functionality of a BOOM design.
+
+.. note:: This work is highly experimental. We hope to integrate this into FireSim in a cleaner fashion at a later point.
```

Comment:
You can drop the last sentence here, if you like, as i think it's selling the work short.

Reply:
I think it would be nice to have it automatically find and add the `dtb` and `rom` into the simulation instead of having to copy it over, hence the sentence. So I don't mind keeping it.